tern/formats/html/generator.py

```diff
@@ -321,16 +321,22 @@ def get_report_dict(image_obj_list):
 
 
 class HTML(generator.Generate):
-    def generate(self, image_obj_list, print_inclusive=False):
+    def generate(self, image_obj_list, spdx_version: str, print_inclusive=False):
         '''Given a list of image objects, create a html report
         for the images'''
+        if spdx_version is not None:
+            logger.warning("The SPDX version parameter is not supported for HTML.")
+
         report_dict = get_report_dict(image_obj_list)
         report = create_html_report(report_dict, image_obj_list)
         return report
 
-    def generate_layer(self, layer):
+    def generate_layer(self, layer, version: str):
```

Comment:
Same comment as above. Update to `spdx_version`.

Reply:
fixed